src/App.tsx

```diff
@@ -1,7 +1,42 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+  wasSomethingPressed: boolean;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+    wasSomethingPressed: false,
```

Comment:
The 'wasSomethingPressed' state is not necessary, you can use 'pressedKey' to determine if a key was pressed or not. You can simplify your code by removing this state variable 😊